src/VueGapi/GoogleAuthService.js

```diff
@@ -222,8 +224,16 @@ export default class GoogleAuthService {
   refreshToken() {
     if (!this.authInstance) throw new Error('gapi not initialized')
     const GoogleUser = this.authInstance.currentUser.get()
-    GoogleUser.reloadAuthResponse().then((authResult) => {
-      this._setStorage(authResult)
+    return new Promise((res, rej) => {
+      GoogleUser.reloadAuthResponse()
+        .then((authResult) => {
+          this._setStorage(authResult)
+          res()
+        })
+        .catch((error) => {
+          console.error(error)
+          rej(error)
+        })
```

Comment:
Any reason to construct a new `Promise`? I think we could just return `GoogleUser.reloadAuthResponse()`'s result:

```suggestion
    return GoogleUser.reloadAuthResponse().then((authResult) => {
      this._setStorage(authResult)
```

It might also be useful to resolve with `authResult`?

Reply:
Agreed. Was trying to keep the current design of `GoogleAuthServive.js` where we've used `return new Promise( .. );`
Have made the required changes now.